api/src/data_migration/models/full_model.sql

```diff
@@ -0,0 +1,59 @@
+MODEL (
+  name api.opportunity_sqlmesh,
+  kind FULL,
+  grain opportunity_id,
+  allow_partials true
+
+--  columns (
+--    opportunity_id BIGINT,
+--    opportunity_number TEXT,
+--    opportunity_title TEXT,
+--    agency TEXT,
+--    opportunity_category_id INTEGER REFERENCES api.lk_opportunity_category(opportunity_category_id),
+--    category_explanation TEXT,
+--    is_draft BOOL,
+--    revision_number INTEGER,
+--    modified_comments TEXT,
+--    publisher_user_id TEXT,
+--    publisher_profile_id BIGINT,
+--  ),
+
+--  @cron "*/5 * * * *"
+
+--  kind SCD_TYPE_2_BY_COLUMN (
+--    unique_key opportunity_id,
+--    columns [oppnumber, opptitle, owningagency, oppcategory, category_explanation,
+--             is_draft, revision_number, modified_comments, publisheruid, publisher_profile_id]
+--  )
```

Comment:
Do these comments tell SQLmesh how to configure the table?

Or would they need to be uncommented in order to apply?